addons/mail/static/tests/chatter/web/chatter.test.js

```diff
@@ -630,6 +631,53 @@ test("schedule activities on draft record should prompt with scheduling an activ
     await waitForSteps(["mail.activity.schedule"]);
 });
 
+test("schedule an activity on record should refresh parent view when hasParentReloadOnActivityChanged is true", async () => {
+    const wizardOpened = new Deferred();
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({ name: "Test" });
+    mockService("action", {
+        doAction(action, options) {
+            if (action.res_model === "res.partner") {
+                return super.doAction(...arguments);
+            } else if (action.res_model === "mail.activity.schedule") {
+                asyncStep("mail.activity.schedule");
```

Comment:
Also asyncStep is deprecated in favor of just step now (but I'd recommend writing a test that doesn't require either in this case)